jekyll/_cci2/use-the-cli-for-config-policy-management.adoc

```diff
@@ -0,0 +1,213 @@
+
+```bash
+mkdir ./config-policies
+```
+
+- Setup a `.circleci/config.yml` to push policies on commits to `main` and show a diff otherwise
+```yaml
+version: 2.1
+
+orbs:
+  circleci-cli: circleci/circleci-cli@0.1.9
+
+workflows:
+  main-workflow:
+    jobs:
+      - diff-policy-bundle:
+          context: [ security-operations ]
+          filters:
+            branches:
+              ignore: main
+      - push-policy-bundle:
+          context: [ security-operations ]
+          filters:
+            branches:
+              only: main
+
+jobs:
+  diff-policy-bundle:
+    executor: circleci-cli/default
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Diff policy bundle
+          command: circleci policy diff ./config --owner-id $OWNER_ID
+
+  push-policy-bundle:
+    executor: circleci-cli/default
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Push policy bundle
+          command: circleci policy push ./config --no-prompt --owner-id $OWNER_ID
+```
+
+Let us break down the previous config:
+
+The following orb makes the `circleci-cli/default` executor available to our jobs
+
+```yaml
+orbs:
+  circleci-cli: circleci/circleci-cli@0.1.9
+```
+
+We then declare two jobs: `diff-policy-bundle` and `push-policy-bundle` to run the policy diff and push commands respectively.
+
+Note that `$OWNER_ID` is an environment variable setup in project settings that is simply your organization id.
+Your organization id is a uuid value that can be found on the organization settings page. 
+
+```yaml
+jobs:
+  diff-policy-bundle:
+    executor: circleci-cli/default
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Diff policy bundle
+          command: circleci policy diff ./config --owner-id $OWNER_ID
+
+  push-policy-bundle:
+    executor: circleci-cli/default
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Push policy bundle
+          command: circleci policy push ./config --no-prompt --owner-id $OWNER_ID
+```
+
+We declare a workflow to run the diff job when not on branch `main` and the push job only on branch `main`
+
+```yaml
+workflows:
+  main-workflow:
+    jobs:
+      - diff-policy-bundle:
+          context: [ security-operations ]
+          filters:
+            branches:
+              ignore: main
+      - push-policy-bundle:
+          context: [ security-operations ]
+          filters:
+            branches:
+              only: main
+```
+
+Note the context for each job: `security-operations`. This context name is arbitrary, however a context is needed to authenticate the CLI. The context must declare an environment variable `CIRCLECI_CLI_TOKEN` that will be used by the CLI.
+
+We recommend creating a bot account for pushing policies and to use its associated CircleCI Token. The context should be restricted to groups that are responsible for managing policies. See Restricted Contexts.
```

Comment:
The `See Restricted Contexts` text should probably be hyperlinked to contexts page.